app/_locales/ko/messages.json

```diff
@@ -488,10 +605,16 @@
   "getHelp": {
     "message": "도움말"
   },
+  "getStarted": {
+    "message": "시작하기"
+  },
   "greaterThanMin": {
     "message": "$1 이상이어야 합니다.",
     "description": "10진수 입력으로 hex값 입력을 도와줍니다"
   },
+  "happyToSeeYou": {
+    "message": "We’re happy to see you."
```

Comment:
Please translate this one too.

Reply:
I had translate this literally, but it got awkward. (`우린 당신을 만나서 행복합니다.`)

Why don't you just leave it there instead of translating it? It is natural enough to see such english sentence in Korea.

Reply:
Yes you're right. Leave this sentence then :)